_docker/drupal/drupal-filesystem/web/modules/custom/rhd_assemblies/src/Service/WordpressApi.php

```diff
@@ -366,10 +383,10 @@ private function getContentMedia($ids) {
    *   If successful, this returns an array of category links and names indexed
    *   by the category ID. Otherwise, this returns an empty array.
    */
-  private function getContentCategoryLinks($categories) {
+  private function getContentCategoryLinks(array $categories) {
```

Comment:
I verified that this will actually be an array even if only 1 category is referenced from a Wordpress post. You can see that via the JSON API in the browser